### ISSUE TITLE
[BUG] False positive on

### ISSUE THREAD

**Issue by reteps** [yeonjuan/html-eslint#392]
**Describe the bug**

- Template expressions are considered in HTML attribute duplicates

```
                <input
                    name="{{name}}"
                    type="text"
                    autocomplete="off"
                    autocorrect="off"
                    class="form-control big-o-input-input {{#parse_error}}has-validation is-invalid{{/parse_error}}"
                    {{#parse_error}}aria-invalid="true"{{/parse_error}}
                    size="{{size}}"
                    {{^editable}}disabled{{/editable}}
                    {{#raw_submitted_answer}}value="{{raw_submitted_answer}}"{{/raw_submitted_answer}}
                    placeholder="{{placeholder}}"
                    {{#suffix}}aria-describedby="pl-big-o-input-{{uuid}}-suffix"{{/suffix}}
                    {{#aria_label}}aria-label="{{aria_label}}"{{/aria_label}}
                    {{^aria_label}}aria-labelledby="pl-big-o-input-{{uuid}}-label"{{/aria_label}}
                />
```

```
   18:83  error  The attribute '{{/aria_label}}' is duplicated  @html-eslint/no-duplicate-attrs
```
**To Reproduce**

```
  {
    files: ['file.mustache'],
    rules: {
      '@html-eslint/no-duplicate-attrs': 'error',
    },
    languageOptions: {
      parser: htmlParser,
      parserOptions: {
        templateEngineSyntax: htmlParser.TEMPLATE_ENGINE_SYNTAX.HANDLEBAR,
      },
    },
  },
```

**Expected behavior**

That the closing handlebar isn't considered an attribute.

**Additional context**

**Comment by yeonjuan** [comment]
@reteps Thanks for reporting this bug. I'll fix it.

**Comment by reteps** [comment]
Thank you!!
